### PR TITLE
Str data unavailable 161207

### PR DESCRIPTION
**Pull request by therearesomewhocallmetim**

### CODE REVIEW

File: strings.txt [hunk 1]
```diff
@@ -20854,3 +20854,7 @@
   [traffic_update_app_message]
     en = To display traffic data, the application must be updated.
     ru = Для отображения пробок необходимо обновить приложение.
+
+  [traffic_data_unavailable]
```
Comment (mpimenov) [inline strings.txt]: s/unavailable/not available/g

**Comment by alexzatsepin** [comment]
LGTM

File: strings.txt (hunk 1 above)
Comment (therearesomewhocallmetim) [inline strings.txt]: Мотивация?

Comment (mpimenov) [inline strings.txt]: https://books.google.com/ngrams/graph?content=data+is+unavailable%2C+data+is+not+available&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cdata%20is%20unavailable%3B%2Cc0%3B.t1%3B%2Cdata%20is%20not%20available%3B%2Cc0

Comment (mpimenov) [inline strings.txt]: Я не вижу в текущем варианте слова "временно".

Comment (mpimenov) [inline strings.txt]: Для любителей трендов есть уточнение: https://books.google.com/ngrams/graph?content=data+unavailable%2Cdata+not+available&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cdata%20unavailable%3B%2Cc0%3B.t1%3B%2Cdata%20not%20available%3B%2Cc0

**Comment by therearesomewhocallmetim** [comment]
ПТАЛ

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**